wallet/how-to/use-non-evm-networks/starknet/create-a-simple-starknet-dapp.md

```diff
@@ -105,23 +128,38 @@ This object can control the connection process, including:
 You can configure these options as follows:
 
 ```typescript title="App.tsx"
-handleConnect({ modalMode: "alwaysAsk", modalTheme: "dark" });
+// Default connection.
+handleConnect()
+
+// Always show modal.
+handleConnect({ modalMode: "alwaysAsk" })
+
+// Never show modal.
+handleConnect({ modalMode: "neverAsk" })
+
+// Dark theme modal.
+handleConnect({
+  modalMode: "alwaysAsk",
+  modalTheme: "dark",
+})
+
+// Light theme modal.
+handleConnect({
+  modalMode: "alwaysAsk",
+  modalTheme: "light",
+})
 ```
 
-### 3.3. Create an `AccountInterface`
-
-After connecting to MetaMask, the account instance is present in the returned object from the `connect` function.
+### 3.3. Handle disconnection
```

Comment:
This can probably be combined with the previous step when refactored according to my suggestion.